src/components/IssueForm/utils.ts

```diff
@@ -301,7 +299,7 @@ const getProjectFromValues = (values: FormValidationSchema): Project["id"] => {
 };
 
 const getAssigneeOptions = (members?: MemberType[]) => {
-  if (!Array.isArray(members) || !size(members)) {
+  if (!Array.isArray(members) || !members.length) {
```

Comment:
This can be simplified to,
```
return (members ?? []).map((member) => {
    const label = createElement(Member, {
      key: member.id,
      name: getFullName(member),
      avatarUrl: get(member, ["avatarUrl"]),
    });
    return getOption(member.id, label, getFullName(member));
  });
  ```

Reply:
simplified